arm-relay/2016-07-01/swagger/relay.json

```diff
@@ -0,0 +1,1930 @@
+        }
+      },
+      "description": "The response of the List WcfRelays operation."
+    },
+    "WcfRelay": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/WcfRelayProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of WcfRelays Resource."
+    },
+    "WcfRelayProperties": {
+      "properties": {
+        "relayType": {
+          "type": "string",
+          "description": "WCFRelay Type.",
+          "enum": [
+            "NetTcp",
+            "Http"
+          ],
+          "x-ms-enum": {
+            "name": "Relaytype",
+            "modelAsString": true
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time the WCFRelay was created."
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time the namespace was updated."
+        },
+        "listenerCount": {
+          "format": "int32",
+          "type": "integer",
+          "readOnly": true,
+          "description": "The number of listeners for this relay. min : 1 and max:25 supported"
+        },
+        "requiresClientAuthorization": {
+          "type": "boolean",
+          "description": "true if client authorization is needed for this relay; otherwise, false."
+        },
+        "requiresTransportSecurity": {
+          "type": "boolean",
+          "description": "true if transport security is needed for this relay; otherwise, false."
+        },
+        "isDynamic": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "true if the relay is dynamic; otherwise, false."
+        },
+        "userMetadata": {
+          "type": "string",
+          "description": "usermetadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."
+        }
+      },
+      "description": "Properties of the WcfRelay Properties."
+    },
+    "NamespaceListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Namespace"
+          },
+          "description": "Result of the List Namespace operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Namespaces"
+        }
+      },
+      "description": "The response of the List Namespace operation."
+    },
+    "Namespace": {
```

Comment:
Namespace is a tracked resource. PATCH and GET operations should be exposed for this resource.